pkg/controllers/management/drivers/base_driver.go

```diff
@@ -114,7 +114,7 @@ func (d *BaseDriver) stage() error {
 		return err
 	}
 
-	tempFile, err := ioutil.TempFile("", "machine-driver")
+	tempFile, err := ioutil.TempFile("", "kontainer-machine-driver")
```

Comment:
I don't think it is the right thing to do. Both kontainer-engine and node-driver will use this logic. It is meaningless to change the temp file name to `kontainer-machine-driver`